var/spack/repos/builtin/packages/salmon/package.py

```diff
@@ -55,10 +56,18 @@ class Salmon(CMakePackage):
     depends_on("libgff")
     depends_on("pkgconfig")
     depends_on("curl", when="@0.14.1:")
+    depends_on("htslib", when="@1.10.2")
+
+    patch("fix_hts.patch", when="@1.10.2")
```

Comment:
Did you report this upstream or are they already aware of it separately and plan to fix it in the next release?

Reply:
https://github.com/COMBINE-lab/salmon/issues/866